local-cli/templates/HelloNavigation/dependencies.json

```diff
@@ -1,3 +1,3 @@
 {
-  "react-navigation": "1.0.0-beta.5"
+  "react-navigation": "1.0.0-beta.9"
```

Comment:
Could you update this to beta.11, and ensure that everything still works as you expect? :)